py3status/modules/wifi.py

```diff
@@ -73,6 +73,8 @@ class Py3status:
     signal_bad = 29
     signal_degraded = 49
     use_sudo = False
+    _ssid = None
+    _max_bitrate = 0
```

Comment:
I only use this netbook for Netflix and other little things.
Got this in my i3/bar config.
```
set $p3 ~/src/py3status/py3status
set $i3 ~/.config/i3

bar {
        status_command py3status -i $p3/modules -i $i3/modules -i /tmp/modules -c $i3/py3status.conf -l /tmp/py3status_log.log
```
I fix here on computer, then I scp `wifi.py` over to the netbook and restart i3wm.

py3status is installed and build iirc.